### DISCUSSION TITLE
How to connect to the same existing Database from another computer or server

### DISCUSSION THREAD

**Discussion by ShokoufehSimab** [mixcore/mix.core#350] [Q&A]
Hi,
First thanks for your answers,
I have a problem connecting to an existed database. 
I have initialized mixcore 's database on the remote server and DB was successfully created and now when I tried to connect to that server database with another computer I have got a "Cannot init culture" error on the init page.

**Comment by Smilefounder** [comment]
Hi @ShokoufehSimab , I think at the moment the best way is to download all of the source code from the remote server to another computer (including appsetting.json) then run **WITHOUT** re-init step.

**Reply by Smilefounder** [reply]
@nhathoang989 kindly share your thought if there are any other ideas.
